pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -676,3 +679,17 @@ message RemoveImageRequest {
 }
 
 message RemoveImageResponse {}
+
+message NetworkConfig {
+    optional string podCIDR = 1;
```

Comment:
Please use CamelCase for field names and add doc comments for it, e.g. `pod_cidr`.